pom.xml

```diff
@@ -118,13 +118,52 @@
 			<artifactId>matrix-project</artifactId>
 			<scope>test</scope>
 		</dependency>
+        <dependency>
```

Comment:
This file uses tab-based indentation. The added lines use space-based indentation. For consistency with the rest of the file, the added lines should use tab-based indentation.